### PR TITLE
cli: implement help <command>

### PR DESCRIPTION
**Pull request by sergiusens**
Allow for calling `snapcraft help <comand>.

Closes: #1699
Signed-off-by: Sergio Schvezov <sergio.schvezov@canonical.com>

- [x] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [x] If this is a bugfix. Have you checked that there is a bug report open for the issue you are trying to fix on [bug reports](https://bugs.launchpad.net/snapcraft)?
- [x] If this is a new feature. Have you discussed the design on the [forum](https://forum.snapcraft.io)?
- [x] Have you successfully run `./runtests.sh static`?
- [x] Have you successfully run `./runtests.sh unit`?

-----

### CODE REVIEW

File: snapcraft/cli/_command_group.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
```
Comment (elopio) [inline snapcraft/cli/_command_group.py]: 2018

File: snapcraft/cli/_runner.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
```
Comment (elopio) [inline snapcraft/cli/_runner.py]: 2018

File: snapcraft/cli/_command_group.py (hunk 1 above)
Comment (sergiusens) [reply snapcraft/cli/_command_group.py]: the changes here are negligible enough to not change the copyright

File: snapcraft/cli/_runner.py (hunk 2 above)
Comment (sergiusens) [reply snapcraft/cli/_runner.py]: the changes here are negligible enough to not change the copyright

**Comment by sergiusens** [comment]
thanks for the review, I will work on the changes.

**Pull request merged**